Makefile

```diff
@@ -168,8 +168,12 @@ esti: ## run esti (system testing)
 
 test: test-go test-hadoopfs  ## Run tests for the project
 
-test-go: gen			# Run parallelism > num_cores: most of our slow tests are *not* CPU-bound.
-	$(GOTEST) -count=1 -coverprofile=cover.out -race -cover -failfast --parallel="$(GOTEST_PARALLELISM)" $(GO_TEST_MODULES)
+make-ui-dir:
+	mkdir ./webui/dist
+	touch ./webui/dist/index.html
```

Comment:
Not sure we would like it in the Makefile - or alternative enable specific step that will just generate the Go files that will skip the node/webui stuff.